### PR TITLE
Update target framework for All metapackage

### PR DESCRIPTION
**Pull request by Shane32**
- Allows All metapackage to be used for ASP.NET Core 2.1
- Serializer and MicrosoftDI is already included within Transports.AspNetCore

### CODE REVIEW

File: src/All/All.csproj [hunk 1]
```diff
@@ -17,8 +17,6 @@
 
     <PackageReference Include="GraphQL.MemoryCache" Version="$(GraphQLVersion)" />
     <PackageReference Include="GraphQL.DataLoader" Version="$(GraphQLVersion)" />
-    <PackageReference Include="GraphQL.MicrosoftDI" Version="$(GraphQLVersion)" />
```
Comment (sungam3r) [inline src/All/All.csproj:20]: I am curious here. Create simple console net5/net6 app, then reference this metapackage. What version (I mean what TFM) of GraphQL.MemoryCache/GraphQL dlls will you see in output folder? netstandard2.0 of net5/net6 ?

Comment (Shane32) [reply src/All/All.csproj:20]: I created a console app only referencing the All meta-package.  I then verified that the DLLs are the appropriate version with a binary file comparison.  Specifically, GraphQL.SystemTextJson (pulled from nuget) is the net6 version (of version 6.0.0-preview-602) and Transports.AspNetCore (pulled from the local build) is the net5 version.  I did not check other files.

**Comment by Shane32** [comment]
I did notice that Transports.AspNetCore does not include GraphQL.NewtonsoftJson because that would bump the Newtonsoft.Json version unnecessarily.  So I included it here for .NET Standard 2.0 projects (which includes .NET Core 2.1).

File: src/All/All.csproj (hunk 1 above)
Comment (sungam3r) [reply src/All/All.csproj:20]: OK. So TFM resolution works "globally" and is not constrained by consuming package.

**Review by sungam3r — APPROVED** [review]

**Pull request merged**